articles/azure-app-configuration/use-key-vault-references-spring-boot.md

```diff
@@ -156,12 +156,29 @@ To add a secret to the vault, you need to take just a few additional steps. In t
 
 1. Create an environment variable called **APP_CONFIGURATION_ENDPOINT**. Set its value to the endpoint of your App Configuration store. You can find the endpoint on the **Access Keys** blade in the Azure portal. Restart the command prompt to allow the change to take effect.
 
-1. Open *bootstrap.properties* in the *resources* folder. Update this file to use the **APP_CONFIGURATION_ENDPOINT** value. Remove any references to a connection string in this file.
-
+1. If your application is using a *bootstrap.properties*, open it in the *resources* folder. Update this file to use the **APP_CONFIGURATION_ENDPOINT** value. Remove any references to a connection string in this file.
+    #### [.properties](#tab/properties)
```

Comment:
```suggestion
### [yaml](#tab/yaml)

    ```yaml
    spring:
      cloud:
        azure:
          appconfiguration:
            stores:
              - endpoint: ${APP_CONFIGURATION_ENDPOINT}
    ```

### [properties](#tab/properties)

    ```properties
    spring.cloud.azure.appconfiguration.stores[0].endpoint= ${APP_CONFIGURATION_ENDPOINT}
    ```

---

> [!NOTE]
> You can also use the [Spring Cloud Azure global configurations](../developer/java/spring-framework/authentication) to connect to Key Vault.
```

Yaml should probably be first, and the note links to all valid authentication methods.